docs/content/latest/api/ysql/datatypes/type_datetime/operators/test-date-time-division-overloads.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Test the date-time division overloads [YSQL]
+headerTitle: Test the date-time division overloads
+linkTitle: Test division overloads
+description: Presents code that tests the date-time division overloads. [YSQL]
+menu:
+  latest:
+    identifier: test-date-time-division-overloads
+    parent: date-time-operators
+    weight: 50
+isTocNested: true
+showAsideToc: true
+---
+
+qqq
```

Comment:
What's this?

Reply:
Ad hoc note to slef to fix/review. Fixed now.